libs/sdk/src/lib/helpers/kin-to-quarks.ts

```diff
@@ -1,6 +1,7 @@
 import BigNumber from 'bignumber.js'
+import { addDecimals } from './add-decimals'
 
 export function kinToQuarks(amount: string): BigNumber {
-  const b = new BigNumber(amount).decimalPlaces(5, BigNumber.ROUND_DOWN)
-  return b.multipliedBy(1e5)
+  console.warn(`Deprecated method: use 'addDecimals(amount, 5)' instead.`)
```

Comment:
Is it possible to use the `Logger` here, or maybe just add some namespacing here, something like:

```js
console.warn(`[KinToQuarks] Deprecated method: use 'addDecimals(amount, 5)' instead.`)
```